extism_pdk.go

```diff
@@ -77,6 +77,25 @@ func Input() []byte {
 	return loadInput()
 }
 
+func JsonFrom(offset uint64, v any) error {
+	mem := FindMemory(offset)
+	return json.Unmarshal(mem.ReadBytes(), v)
+}
+
+func InputJson(v any) error {
+	return json.Unmarshal(Input(), v)
+}
+
+func OutputJson(v any) error {
```

Comment:
curious if we should name these `XxxJSON` as uppercase JSON a la `UnmarshalJSON` interface functions in the Go stdlib.

Reply:
I it seems like Go libraries typically use `JSON`, but as long as we're consistent I don't mind. `Json` feels slightly more readable.

Reply:
Yea I'm in the same boat. Feels fine as-is, but it might be more familiar to a Go developer and immediately obvious what the function is for if we use JSON. 

I may just switch it out and see...

Reply:
I ended up switching these to the uppercase variant and have added this to the docs too